app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
+  @extend .btn-default;
+}
+
+input:checked + .inputButton, input:checked + .inputButton:active:hover {
   background-color: $btn-success-bg;
   color: $btn-success-color;
-}
 
-input:checked + .radioButton > .help-col {
-  color: #D8D8D8;
+  .help-col {
+    color: #D8D8D8;
+  }
 }
 
 .button-row {
   display: flex;
   flex-direction: row;
-}
 
-@media (max-width: $screen-sm-min) {
-  .button-row {
-    flex-direction: column;
+  .button-item {
+    flex: auto;
+    flex-basis: 0;
+
+    .inputButton {
+      width: 100%;
+      height: 100%;
+    }
   }
 }
 
-.button-row > .button-item {
-  flex: auto;
-  flex-basis: 0;
-}
+.button-grid {
+  display: flex;
+  flex-wrap: wrap;
 
-.button-row > .button-item > label {
-  width: 100%;
-  height: 100%;
+  .button-item {
+    width: 33.33%;
+    padding: 4px;
```

Comment:
Don't set the padding in `px`. Set it in `em`, so that it varies accordingly to the inner font size.